CONTRIBUTING.md

```diff
@@ -0,0 +1,75 @@
+# How to Contribute
+
+Welcome, and thanks for considering contributing! In order to make our
+review/development process easier, we have some guidelines to help you figure
+out how to contribute to Project Lorax.
+
+
+## Getting Started
+
+If you're new to contributing to Mozilla, you may want to read through the
+[Mozilla Webdev Bootcamp](http://mozweb.readthedocs.org/en/latest/), a tutorial
+and reference guide to contributing as a web developer to Mozilla.
+
+
+## Reporting Issues
+
+We use [Bugzilla][] to track issues and bugs for Project Lorax. Any issues
+with the website should be filed under the Shape of the Web component in the
+Websites product.
+
+[Bugzilla]: https://bugzilla.mozilla.org/buglist.cgi?product=Websites&component=Shape of the Web&resolution=---
```

Comment:
For some reason it's not parsing this correctly. Might be the spaces in the URL.